crowbar_framework/app/models/api/node.rb

```diff
@@ -38,6 +39,7 @@ def pre_upgrade
     end
 
     def prepare_repositories
+      save_node_action("preparing repositories")
```

Comment:
How about "upgrading repository configuration"? "preparing repositories" is very generic and to me sound a bit like "doing something" :)